federation/pkg/kubefed/init/init_test.go

```diff
@@ -1444,7 +1444,8 @@ func tlsHandshake(t *testing.T, sCfg, cCfg *tls.Config) error {
 		}
 	}()
 
-	c, err := tls.Dial("tcp", s.Addr().String(), cCfg)
+	addr := strings.Split(s.Addr().String(), ":")
```

Comment:
Thanks for your guidance.
Your are right. When I enable ipv6, this test passes in my environment.

I passed tcp4 to tls.Listen() for the guys who disables ipv4.